charts/netbird/templates/management-deployment.yaml

```diff
@@ -69,10 +69,10 @@ spec:
           {{- end }}
           ports:
             - name: {{ .Values.management.service.name }}
-              containerPort: {{ .Values.management.service.port }}
+              containerPort: {{ .Values.management.service.containerPort }}
```

Comment:
I would use values.management.containerport instead of .Values.management.service.containerPort